onnxruntime/core/graph/model.h

```diff
@@ -51,7 +51,8 @@ class Model {
                  const IOnnxRuntimeOpSchemaRegistryList& local_registries,
                  const std::unordered_map<std::string, int>& domain_to_version,
                  const std::vector<ONNX_NAMESPACE::FunctionProto>& model_specific_functions,
-                 const logging::Logger& logger);
+                 const logging::Logger& logger,
+                 const std::vector<ONNX_NAMESPACE::FunctionProto>* model_functions = nullptr);
```

Comment:
What is the difference between this and the parameter two lines above?

Reply:
Nit: The comment above "construct model from scratch" seems to imply that this model is constructed by mutation: e.g., the graph is not a constructor parameter, but it is constructed subsequently (via mutation). I wonder if the same would be fine for functions also.

Reply:
> What is the difference between this and the parameter two lines above?
this is the result of a bad merge. Fixed it.